### PR TITLE
doc reorder mr, stream to stream, mr

### PR DESCRIPTION
**Pull request by karthikeyann**
Doc reorder mr, stream to stream, mr

### CODE REVIEW

**Review by davidwendt — APPROVED** [review]

File: cpp/src/reductions/minmax.cu [hunk 1]
```diff
@@ -60,7 +60,6 @@ struct minmax_pair {
  * @param d_in input iterator
  * @param num_items number of items to reduce
  * @param binary_op binary operator used to reduce
- * @param mr Device resource used for result allocation
```
Comment (mythrocks) [inline cpp/src/reductions/minmax.cu:63]: Would it make sense to document `T`, `OutputType`, as part of this change?

**Review by mythrocks — APPROVED** [review]
:+1 
Some minor suggestions to document some peripheral function parameters.

File: cpp/src/reductions/minmax.cu (hunk 1 above)
Comment (karthikeyann) [reply cpp/src/reductions/minmax.cu:63]: `T` template param is not needed in this struct. removed it.
added documentation for `OutputType`.

File: cpp/src/binaryop/compiled/binary_ops.hpp [hunk 2]
```diff
@@ -120,8 +120,8 @@ std::unique_ptr<column> binary_operation(
  * @param lhs         The left operand string column
  * @param rhs         The right operand string column
  * @param output_type The desired data type of the output column
- * @param mr          Device memory resource used to allocate the returned column's device memory
```
[the remark this answers is not in the document]
Comment (karthikeyann) [reply cpp/src/binaryop/compiled/binary_ops.hpp:123]: done

**Comment by karthikeyann** [comment]
@gpucibot merge

**Pull request merged**